spaceR/spaceR/Config/AppDelegate.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  AppDelegate.swift
+//  spaceR
+//
+//  Created by Kieran Woodrow on 2022/02/21.
+//
+
+import UIKit
+
+@main
+class AppDelegate: UIResponder, UIApplicationDelegate {
+
+    func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+        // Override point for customization after application launch.
```

Comment:
A general rule of thumb is that you are more than welcome to have comments if it will help you work on your feature branches, but once you are ready to commit changes to develop then you'll need to remove the comments